quirks/system76-nvidia-quirks-thelio-massive-b1

```diff
@@ -0,0 +1,12 @@
+Section "Quirk"
+    Identifier "system76-0-thelio-massive-b1"
+    Handler "nvidia-driver-465|nvidia-driver-460"
+    Match "sys_vendor" "System76, Inc.|System76"
```

Comment:
We're checking which one of these vendors is actually in the DMI table for massive-b1 firmware, since they should all be the same for a single product with a single firmware image.

Reply:
Verified the vendor is set to System76 on Massive-b1 removed check for the other one.